src/components/UserInfo.js

```diff
@@ -1,22 +1,35 @@
 export default class UserInfo {
-  constructor({ profileName, profileJob }) {
+  constructor({ profileName, profileJob, profileAvatar, isLiked }) {
     this._profileName = document.querySelector(profileName);
     this._profileJob = document.querySelector(profileJob);
+    this._profileAvatar = document.querySelector(profileAvatar);
+    this._isLiked = document.querySelector(isLiked);
   }
 
   getUserInfo() {
     //returns user information
     return {
       profileName: this._profileName.textContent,
       profileJob: this._profileJob.textContent,
+      profileAvatar: this._profileAvatar.src,
     };
   }
 
-  setUserInfo({ profileName, profileJob }) {
+  setUserInfo({ profileName, profileJob, profileAvatar, isLiked }) {
     this._profileName.textContent = profileName;
     this._profileJob.textContent = profileJob;
+    this._profileAvatar.src = profileAvatar;
+    this._isLiked = isLiked;
```

Comment:
`this._isLiked = isLiked;` is not needed inside `UserInfo` because it's not about the `profile` data